packages/scss/src/components/inputs/textfield/_textfield.compact.scss

```diff
@@ -43,8 +43,8 @@
 	}
 
 	.textfield-suffix {
-		top: _component("textfield.compact.padding");
-		right: _component("textfield.compact.padding");
+		top: _component('textfield.compact.suffix.top');
```

Comment:
Why add a new variable here? It seemed logical that the position was set to it's parent padding. (and .5rem seems more centered than .6rem for the top variable)

Reply:
✔️